src/pageEditor/panes/ModComponentEditorPane.tsx

```diff
@@ -87,26 +88,49 @@ const EditorPaneContent: React.VoidFunctionComponent<{
   );
 };
 
-const ModComponentEditorPane: React.VFC = () => {
-  // Inject the draft mod instance into the page while editing
-  useRegisterDraftModInstanceOnAllFrames();
-
+function useInitialValues(): ModComponentFormState {
```

Comment:
Opportunity to clarify/encourage this pattern, e.g.:
```suggestion
/**
* Returns the active mod component form state. Responds to updates in the editor state for use with triggering rerenders.
*/
function useInitialValues(): ModComponentFormState {
```